From 689a97f52383110bf0da049e8f6294993f4020dd Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Thu, 08 Aug 2013 19:44:02 +0000
Subject: udevd: respect the log-level set in /etc/udev/udev.conf

A regression introduced when we moved to systemd's logging is that the only
way to adjust the log-level of the udev daemon is via the env var, kernel
commandline or the commandline.

This reintroduces support for specifying this in the configuration file.
---
diff --git a/src/udev/udevd.c b/src/udev/udevd.c
index 45ec3d6..7c6c5d6 100644
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -923,7 +923,10 @@ int main(int argc, char *argv[])
         log_set_target(LOG_TARGET_AUTO);
         log_parse_environment();
         log_open();
+
         udev_set_log_fn(udev, udev_main_log);
+        log_set_max_level(udev_get_log_priority(udev));
+
         log_debug("version %s\n", VERSION);
         label_init("/dev");
 
--
cgit v0.9.0.2-2-gbebe
